native/python_package/python_driver/requestprocessor.py

```diff
@@ -109,11 +118,24 @@ def _return_error(self, filepath: AnyStr='', status: AnyStr='error') -> None:
         :param status: error type, 'error' or 'fatal'
         """
 
+        if status == 'fatal':
+            ret_ast = None
+        elif ast:
+            ret_ast = ast
+        else:
+            # Empty modules are still modules
```

Comment:
The function produces and error result and sends it; it doesn't return anything as function return value. Tried returning the empty AST before in the case of the empty module but unfortunately the SDK at some point turns and empty AST into a `fatal` error which is actually as defined (fatal error is the lack of an AST) but undesirable in this case.

Actually, now that I think again about it, empty code shouldn't be an error - but it will still produce this "module only" AST since in Python eve binary modules can have meaning (like `__init__.py` in this case). So I'll change that part.